Enigmatry.Entry.AspNetCore.Authorization/Attributes/UserHasPermissionAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Enigmatry.Entry.AspNetCore.Filters;
+using Microsoft.ApplicationInsights.AspNetCore.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Logging;
+
+namespace Enigmatry.Entry.AspNetCore.Authorization.Attributes;
+public sealed class UserHasPermissionAttribute : AuthorizeAttribute, IAuthorizationFilter
+{
+    public string Permissions { get; }
```

Comment:
For the sake of consistency mostly. The existing Attributes in asp.net also use a comma-sperated string (like: [Authorize(Roles = "HRManager,Finance")].

Reply:
Ok, I'm fine with comma. However, I would like to avoid passing in the string of comma separated roles down the call stack (i.e. IAuthorizationProvider), and split them immediately in the attribute.

Reply:
Yes, it is currently implemented that way.